package.json

```diff
@@ -15,7 +15,9 @@
     "generate:videos": "node scripts/build-newsroom-videos.js",
     "generate:tools": "node scripts/build-tools.js",
     "test": "echo \"No tests configured yet\"",
-    "release": "echo \"No release to npm for this project\""
+    "release": "echo \"No release to npm for this project\"",
+    "lint:check": "eslint . --max-warnings=0",
```

Comment:
```suggestion
    "lint": "eslint . --max-warnings=0",
```

no need to say `check`, we do it in other projects, so user just run `npm run lint`